docs/visualize/aggregations.asciidoc

```diff
@@ -48,56 +63,84 @@ Parent pipeline aggregations assume the bucket aggregations are ordered and are
 
 You can also nest these aggregations. For example, if you want to produce a third derivative.
 
-{ref}/search-aggregations-pipeline-bucket-script-aggregation.html[Bucket script]:: Executes a script that performs computations for each bucket that specifies metrics in the parent multi-bucket aggregation.
+[cols="2*"]
+|===
+|{ref}/search-aggregations-pipeline-bucket-script-aggregation.html[`Bucket script`]
+|Executes a script that performs computations for each bucket that specifies metrics in the parent multi-bucket aggregation.
 
-{ref}/search-aggregations-pipeline-cumulative-sum-aggregation.html[Cumulative sum]:: Calculates the cumulative sum of a specified metric in a parent histogram.
+|{ref}/search-aggregations-pipeline-cumulative-sum-aggregation.html[`Cumulative sum`]
+|Calculates the cumulative sum of a specified metric in a parent histogram.
 
-{ref}/search-aggregations-pipeline-derivative-aggregation.html[Derivative]:: Calculates the derivative of specific metrics.
+|{ref}/search-aggregations-pipeline-derivative-aggregation.html[`Derivative`]
+|Calculates the derivative of specific metrics.
 
-{ref}/search-aggregations-pipeline-movavg-aggregation.html[Moving avg]:: Slides a window across the data and emits the average value of the window.
+|{ref}/search-aggregations-pipeline-movavg-aggregation.html[`Moving avg`]
+|Slides a window across the data and emits the average value of the window.
 
-{ref}/search-aggregations-pipeline-serialdiff-aggregation.html[Serial diff]:: Values in a time series are subtracted from itself at different time lags or periods.
+|{ref}/search-aggregations-pipeline-serialdiff-aggregation.html[`Serial diff`]
+|Values in a time series are subtracted from itself at different time lags or periods.
```

Comment:
Suggest:

Subtracts values in a time series at different time lags or periods.